libsplinter/Cargo.toml

```diff
@@ -137,6 +138,7 @@ rest-api = [
     "percent-encoding",
 ]
 rest-api-actix = ["actix", "actix-http", "actix-web", "actix-web-actors"]
+rest-api-auth = ["auth"]
```

Comment:
This is a bit different, though, since it isn't just guarding some endpoints; it guards some functionality of the REST API itself. I added this feature on the assumption that enabling the `auth` module is distinct from enabling auth for the REST API. This could be useful if we use the `auth` module for something other than the REST API. If this isn't something that seems necessary, though, I can remove this feature.

Reply:
Removed this feature